jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/BuildStepsRunner.java

```diff
@@ -55,7 +89,13 @@ public static BuildStepsRunner forBuildImage(BuildConfiguration buildConfigurati
       throws CacheDirectoryCreationException {
     return new BuildStepsRunner(
         BuildSteps.forBuildToDockerRegistry(
-            buildConfiguration, getCacheInitializer(buildConfiguration)));
+            buildConfiguration, getCacheInitializer(buildConfiguration)),
+        String.format(
```

Comment:
I don't see any "%s" in `STARTUP_MESSAGE_PREFIX_FOR_DOCKER_REGISTRY`. Is `String.format()` working as intended?

Reply:
Oops, this was supposed to be unwrapped